pkgs/by-name/ni/niri/package.nix

```diff
@@ -64,6 +64,22 @@ rustPlatform.buildRustPackage rec {
 
   LIBCLANG_PATH = "${libclang.lib}/lib";
 
+  passthru.providedSessions = ["niri"];
+
+  postInstall = ''
+    mkdir -p $out/share/systemd/user
+    mkdir -p $out/share/wayland-sessions
+    mkdir -p $out/share/xdg-desktop-portal
+
+    cp ${src}/resources/niri-session $out/bin/niri-session
+    cp ${src}/resources/niri.service $out/share/systemd/user/niri.service
+    cp ${src}/resources/niri-shutdown.target $out/share/systemd/user/niri-shutdown.target
+    cp ${src}/resources/niri.desktop $out/share/wayland-sessions/niri.desktop
+    cp ${src}/resources/niri-portals.conf $out/share/xdg-desktop-portal/niri-portals.conf
```

Comment:
Can we take those from a relative path, so that potential patches apply to them?